src/app/Startup.cs

```diff
@@ -53,6 +63,7 @@ public void ConfigureServices(IServiceCollection services)
             {
                 services.AddApplicationInsightsTelemetry(appInsightsKey);
             }
+
```

Comment:
can delete

Reply:
Which part can be deleted, line 64? This comment has line 66 highlighted but there is no code to review. If it is line 64, can you elaborate on why?